PixiEditor/Models/Tools/Tool.cs

```diff
@@ -9,11 +10,13 @@ namespace PixiEditor.Models.Tools
     public abstract class Tool : NotifyableObject
     {
         private bool isActive;
-        private string actionDisplay = "";
-
-        public abstract ToolType ToolType { get; }
+        private string actionDisplay = string.Empty;
+
+        public string Name => GetType().Name.Replace("Tool", string.Empty);

```

Comment:
This shouldn't be a getter, instead of `Name =>` it should be `Name = `, because maybe someone would want to change the name in the constructor